src/main/java/com/solr/clientwrapper/rest/EmployeeController.java

```diff
@@ -66,6 +73,76 @@ public Employee getEmployeeByAddress(@PathVariable String address) {
     private SolrClient solrClient;
     private final static String solrDataName = "employee";
 
+    @GetMapping("/coreStatus/{name}")
+    public String coreStatus(@PathVariable String name) throws SolrServerException, IOException, URISyntaxException, ParserConfigurationException, InterruptedException, TransformerException, org.xml.sax.SAXException {
+
+        HttpSolrClient solrClient = new HttpSolrClient.Builder("http://localhost:8983/solr").build();
```

Comment:
Create solradaptor and call the solr api from there
we should not call any external api from controller

Reply:
Employee Controller was only for testing. Removed it and also the other related classes.